pkg/remote/aws/repository/elbv2_repository.go

```diff
@@ -0,0 +1,38 @@
+package repository
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/aws/aws-sdk-go/service/elbv2/elbv2iface"
+	"github.com/snyk/driftctl/pkg/remote/cache"
+)
+
+type ELBv2Repository interface {
```

Comment:
It would be here `ELBV2Repository` but you're right on the struct below `elbv2Repository`.

Either we keep your convention or we use the one from the API Gateway V2.

What is the idiomatic way in go? I don't remember.